src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -290,9 +290,16 @@ AttestationVerificationResult DefaultDACVerifier::ValidateCertificationDeclarati
     VerifyOrReturnError(CMS_ExtractKeyId(cmsEnvelopeBuffer, skid) == CHIP_NO_ERROR,
                         AttestationVerificationResult::kCertificationDeclarationNoKeyId);
 
-    VerifyOrReturnError(GetCertificationDeclarationCertificate(skid, certificateSpan) == CHIP_NO_ERROR,
+    auto err = mAttestationTrustStore->GetCertificationDeclarationCert(skid, certificateSpan);
+    VerifyOrReturnError(err == CHIP_NO_ERROR || err == CHIP_ERROR_NOT_IMPLEMENTED,
                         AttestationVerificationResult::kCertificationDeclarationNoCertificateFound);
 
+    if (err == CHIP_ERROR_NOT_IMPLEMENTED)
+    {
+        VerifyOrReturnError(GetCertificationDeclarationCertificate(skid, certificateSpan) == CHIP_NO_ERROR,
```

Comment:
The name of `GetCertificationDeclarationCertificate` should be changed to `GetDevelopmentCDSigningKey`